app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -266,6 +266,8 @@ def select_release_varies_option(permission_template)
       # rubocop:disable Metrics/CyclomaticComplexity
       # rubocop:disable Metrics/PerceivedComplexity
       def permission_template_update_params
```

Comment:
is there value in a unit test for this change? this method has some existing tests, and it might help future devs to have a clearly defined spec for the behavior in this change.

Reply:
I just added unique test https://github.com/samvera/hyrax/pull/6163/commits/98d9caee49aef84a9d5821926c8fb66991037779 for it.